graphql.md

```diff
@@ -0,0 +1,54 @@
+# GraphQL Development
```

Comment:
The structure/layout of this page is different to the others that I have seen, e.g. https://github.com/nodeshift/nodejs-reference-architecture/blob/master/auth.md.

```
# GraphQL
## Recommended Components
 - Component1
 - Component2
 - ...
## Guidance
```

Perhaps this is not achievable since the GraphQL section can be spread across frontend, backend, tooling, database etc and needs to be broken down further.

```
# GraphQL
## Client
### Recommended Components
 - URQL
 - GraphQL Tag
 - GraphQL Tools
 - GraphQL Code Generator
 - Offix
### Guidance
## Server
### Recommended Components
 - GraphQL Tools
 - GraphQL Code Generator
 - Graphback
 - GraphQL-Express
### Guidance
```

Reply:
The architecture is meant to cover what an Node.js developer needs to know about the topic.  This means that we won't necessarily cover the server side (ie we would not necessarily say use `Apollo` or something like that as it's not Node.js related and depend on what an organization is already using.

On the other hand some of the things you mention under the Server section seem like they could be related.

I'm thinking we still start with the format we see in other docs 
```markdown
# GraphQL
## Recommended Components
 - Component1
 - Component2
 - ...
## Guidance
```

and then in the description for each component explain what it relates to. I had a similar back and forth on that idea for another topic I'm writing up and I think it will work in the end.

Reply:
Thank you for reply. Fully agree. I had plans to update both PR to this format